Source/WebCore/css/typedom/CSSStyleValueFactory.cpp

```diff
@@ -196,9 +196,12 @@ ExceptionOr<Ref<CSSStyleValue>> CSSStyleValueFactory::reifyValue(const CSSValue&
             return Ref<CSSStyleValue> { CSSNumericFactory::em(primitiveValue->doubleValue()) };
         case CSSUnitType::CSS_EXS:
             return Ref<CSSStyleValue> { CSSNumericFactory::ex(primitiveValue->doubleValue()) };
+        case CSSUnitType::CSS_CAP:
+            return Ref<CSSStyleValue> { CSSNumericFactory::cap(primitiveValue->doubleValue()) };
         case CSSUnitType::CSS_CHS:
             return Ref<CSSStyleValue> { CSSNumericFactory::ch(primitiveValue->doubleValue()) };
-        // FIXME: Add CSSNumericFactory::ic
+        case CSSUnitType::CSS_IC:
```

Comment:
Any test coverage for this drive-by fix?

Reply:
WPT is pretty lacking for these in general, I'll make a follow up PR with test coverage for all of those